core/src/main/cpp/external/yahfa/src/HookMain.cpp

```diff
@@ -116,6 +116,10 @@ namespace yahfa {
         switch (sdkVersion) {
             default:
                 LOGE("not compatible with SDK %d", sdkVersion);
+            case __ANDROID_API_FUTURE__:
+                // fallthrough
+            case __ANDROID_API_T__:
+            case 32: // stupid Google
```

Comment:
Maybe this better?
```c++
#ifndef __ANDROID_API_SV2__
#define __ANDROID_API_SV2__ 32
#endif
```

Reply:
I think so, but I don't the if the name should be `SV2`. Maybe it should be `MR1` or so. So I decided to use `32` directly.